.github/workflows/ci.yml

```diff
@@ -0,0 +1,575 @@
+          # FIXME: cust-covfix doesn't support the external target dir
+          # https://github.com/Kogia-sima/rust-covfix/issues/7
+          unset "CARGO_TARGET_DIR"
+          cargo clean
+          # make sure there's no stale coverage artifacts
+          find . -name "*.profraw" -type f -delete
+          find . -name "*.gcda" -type f -delete
+          # RUSTFLAGS are the cause target cache can't be used here
+          cargo build --verbose --all-features --workspace
+          cargo test --verbose --all-features --no-fail-fast --workspace
+          # coverage with branches
+          grcov . --binary-path ./target/debug/ --source-dir . --output-type lcov --llvm --branch \
+            --ignore-not-existing --ignore "/*" --ignore "tests/*" --output-path lcov-w-branch.info
+          rust-covfix lcov-w-branch.info --output lcov-w-branch-fixed.info
+          # TODO: Enable when Gitlab CI is disabled
+          #codecov --token "$CODECOV_P_TOKEN" --file lcov-w-branch-fixed.info --nonZero
+          # lines coverage
+          grcov . --binary-path ./target/debug/ --source-dir . --output-type lcov --llvm \
+            --ignore-not-existing --ignore "/*" --ignore "tests/*" --output-path lcov-lines.info
+          rust-covfix lcov-lines.info --output lcov-lines-fixed.info
+          # TODO: Enable when Gitlab CI is disabled
+          #codecov --token "$CODECOV_TOKEN" --file lcov-lines-fixed.info --nonZero
+
+### examples
+
+  examples-test:
+    runs-on: ubuntu-latest-8-cores
+    needs: [clippy]
+    defaults:
+      run:
+        shell: bash
+    container:
+      image: paritytech/ci-unified:bullseye-1.73.0
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 100
+
+      - name: Cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          cache-directories: ${{ env.CARGO_TARGET_DIR }}
+
+      - name: Rust Info
+        uses: ./.github/rust-info
+
+      - name: Test Examples
+        env:
+          # Fix linking of `linkme`: https://github.com/dtolnay/linkme/issues/49
+          RUSTFLAGS:                     -Clink-arg=-z -Clink-arg=nostart-stop-gc
+        run: |
+          # run all tests with --all-features, which will run the e2e-tests feature if present
+          scripts/for_all_contracts_exec.sh --path integration-tests --ignore static-buffer -- cargo test \
+            --verbose --all-features --manifest-path {}
+          # run the static buffer test with a custom buffer size
+          cargo clean --manifest-path integration-tests/static-buffer/Cargo.toml
+          INK_STATIC_BUFFER_SIZE=30 cargo test --verbose --manifest-path integration-tests/static-buffer/Cargo.toml --all-features
+
+  examples-contract-build:
+    runs-on: ubuntu-latest
+    needs: [build]
+    defaults:
+      run:
+        shell: bash
+    container:
+      image: paritytech/ci-unified:bullseye-1.73.0
```

Comment:
```suggestion
      image: ${{ env.IMAGE }}
```

Reply:
I think that in this context, environment variables are not being handled